source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRHostConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+//  ADCIOSAdaptiveHostConfig.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "TextBlock.h"
+#import "Image.h"
+#import "HostConfig.h"
+
+using namespace AdaptiveCards;
+
+@interface ACRHostConfig:NSObject
+
+- (CGSize) getImageSize:(std::shared_ptr<Image>) img;
```

Comment:
>std::shared_ptr<Image> [](start = 25, length = 22)

It would be better to pass these by const reference, to avoid making unnecessary copies.